packages/shared/src/components/modals/utils/ActionSuccessModal.tsx

```diff
@@ -55,16 +56,22 @@ export function ActionSuccessModal<T extends AllowedTags>({
             onClick={props.onRequestClose}
             variant={ButtonVariant.Primary}
           />
-          <div className="relative h-full max-h-56 min-h-52 w-full">
-            <Image
-              className="absolute"
-              src={isTablet ? cover : coverDrawer || cover}
-              alt="Success"
-            />
-          </div>
+          {image && (
```

Comment:
Could there be some edge-case where initial load image is undefined, but then shows?
If so you'd get layout shift on the image appearing.

But not sure if image is maybe optional it can work.

Reply:
Alright makes sense, just wasn't sure if we always expect image or not :)